### ISSUE TITLE
XBMC lockup on Pandora Buffering In ACE Skin and loses Vis in Confluence Skin

### ISSUE THREAD

**Issue by neverumind** [rivy/xbmc-script.audio.pandora#15]
xbmc-12.3 for Windows
XBMC ACE Skin
Pandora 2.3.2 Script Rivy (thanks for the work: btw :+1: )
all the ingredients: 
Frodo
Ace Skin
Pandora Config:
Pandora One: Off (free loader)
AQ: High
Skin: Smoke
TArt: Album
Sort Station: ON
Auto start play previous selected station: ON
Display Pandora Logo Full Screen: ON
Use Proxy: OFF
Scrobbling: OFF
Misc config:
Milkdrop Vis ON:  Furniture OFF
Pandora added to favorite as a workaround as Pandora in Full screen will lose the ability (logo icon fades away and can't re-invoke it) to go back to the Main Pandora menu to TU or TD or change station.  In other words, I can not go full screen and then come back. 

Go into Full Screen and Pandora will play one or two songs then quit to last menu, or lock up 'buffering', The work around is to use Key "S" to get to system menu options and go back to favorites and hit Pandora again. 

To summarize:
There is no persistent Pandora GUI to navigate to once I off the GUI.  This caused the station to quit, attempting to go back to Pandora Add-on in Music causes system beep and nada.  Completed XBMC restart to workaround. Next attempt was to add as Favorited to have access to launch.  This worked as expected to bring up main Pandora GUI yet going full screen caused same fullscreen issue, and or getting dumped after one/two songs, or the 'Buffering' Lock preventing left/right navigation or access to up/down navigation in Setting Main menu to try a hot ~ Add-On plugin config / save.

Thanks in advance for the script and especially your effort and interest in making Pandora available in XBMC.

**Comment by neverumind** [comment]
xbmc-12.3 for Windows
 XBMC Skin Confluence
 Pandora Config:
Pandora One: Off 
AQ: High
Skin: Smoke
TArt: Album
Sort Station: ON
Auto start play previous selected station: OFF (changed testing in ACE skin and left OFF for Confluence)
Display Pandora Logo Full Screen: ON
Use Proxy: OFF
Scrobbling: OFF

Misc config:
Milkdrop Vis ON
Setting:Music:Playback: Cross-fade between songs: 10 seconds. Will test this OFF and update w/ comment.

 Pandora playing from GUI is fine.  Milkdrop is in background doing it's thing and the Pandora GUI works as expected. Going fullscreen
 the logo will remain yet after two songs the Milkdrop Vis will disappear leaving a screen burning bright white w/ the logo in upper left. 
 User has to hit "TAB", "Esc" or "I" to navigate back to Vis + Info.  Bug repeats after two songs finish.

 Buttons used in Confluence Skin are ignored or mapped differently.. Ie  The Escape key from Pandora GUI will kill Pandora in Confluence whereas in ACE it Navigated back to Main/Last used menu allowing user to go to settings and configure items or other main menu items. (Music, Setting, Favorites, etc)
 In ACE the "M" button will invoke the Music Menu (FF,Pause,Play,RR, Setting for Viewing Lyrics, Furniture etc) whereas in Confluence it is ignored. Ignored is better IMO.

 Comments:
 Adjusting Cross-fade to OFF and the lose of Visualization ceased until I forced an advance to next song by using keyboard key ">".  After the next song finished the Vis disappeared.
 If white screen is ignored AND music plays as into the Screen Saver timeout the screen will dim (if SS is set to dim) until next song is queued. Repeat, sequence of Bright/Dim then visualization re-initializes.
